api/rpc/account/account.go

```diff
@@ -0,0 +1,357 @@
+package account
+
+import (
+	"fmt"
+	"net/mail"
+
+	context "golang.org/x/net/context"
+
+	pb "github.com/golang/protobuf/ptypes/empty"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"gopkg.in/hlandau/passlib.v1"
+)
+
+const codeLength = 8
+const hash = "$s2$16384$8$1$42JtddBgSqrJMwc3YuTNW+R+$ISfEF3jkvYQYk4AK/UFAxdqnmNFVeUw2gUVXEMBDAng=" // password
+
+// Validate validates SignUpRequest
+func (r *SignUpRequest) Validate() (err error) {
```

Comment:
Don't really see a compelling need for named error return value in validators. It's basically ignored and requires an explicit terminal return statement that's just noise.